src/jade/tabs/widgets/balances.jade

```diff
@@ -83,4 +83,4 @@
                 ) {{ component | rpamount:{tiny_precision: 3} }}
 
     p(ng-show="$route.current.tabName == 'balance'")
-      a.btn.btn-cancel(href="#/trust", l10n) add gateway
+      a.btn.btn-cancel(href="#/xrp", l10n) add gateway
```

Comment:
@annatonger @mrajvanshy Is /xrp page the right page to route to?  Maybe /trust makes more sense in the other spot where we have add gateway link?